agent-config.json

```diff
@@ -12,5 +12,8 @@
     "numStds" : 3,
     "minElements": 40
   },
+	"check-reserve-update-frequency" : {
+		"oracleAgeThresholdSeconds": 86400
+	},
```

Comment:
spaces not tabs por favor
```vim
autocmd FileType javascript setlocal shiftwidth=2 tabstop=2
autocmd FileType json setlocal shiftwidth=2 tabstop=2```

adding this to your vimrc will make life eaassy

Reply:
`expandtab` added for `json`, `javascript`, and `typescript`

More importantly, tabs replaced with spaces.